### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by arminta7** [nicrivard/readwise2directory#3]
Hey, I'm running the script and it seems to be working great until this:

```Fetching tags for 924 highlights in bulk via CSV export...
Opening new Chrome browser window...
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/common/service.py", line 72, in start
    self.process = subprocess.Popen(cmd, env=self.env,
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 947, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1819, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/arminta/Documents/readwise2directory-master/readwise-GET.py", line 1306, in <module>
    fetchTagsIndividuallyOrInBulk() # Function to determine whether to fetch tags individually or in bulk
  File "/Users/arminta/Documents/readwise2directory-master/readwise-GET.py", line 1279, in fetchTagsIndividuallyOrInBulk
    list_HighlightedAt, list_ReadwiseBookId, list_Source, list_Url, list_NumberOfHighlights, list_UpdatedAt, list_HighlightId = runFetchCsvData()
  File "/Users/arminta/Documents/readwise2directory-master/readwise-GET.py", line 788, in runFetchCsvData
    downloadCsvExport(downloadsDirectory, sourceDirectory, readwiseCsvExportFileName)
  File "/Users/arminta/Documents/readwise2directory-master/readwise-GET.py", line 582, in downloadCsvExport
    driver = webdriver.Chrome(chromedriverDirectory, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/chrome/webdriver.py", line 73, in __init__
    self.service.start()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/selenium/webdriver/common/service.py", line 86, in start
    raise WebDriverException(
selenium.common.exceptions.WebDriverException: Message: '' executable may have wrong permissions. Please see https://sites.google.com/a/chromium.org/chromedriver/home```

**Comment by nicrivard** [comment]
Hey @arminta7 - thanks for trying the script and raising this issue! A couple of questions:

1. Can you check you’ve entered the full path to your `chromedriver.exe` file in `readwiseMetadata`? For example, my version of Chrome is v87, and my operating software is Windows, so I downloaded `chromedriver_win32.zip` from [ChromeDriver v87](https://chromedriver.storage.googleapis.com/index.html?path=87.0.4280.88/) and moved the `chromedriver.exe` file to my downloads folder. Then in `readwiseMetadata` I only need to specify the full path as `/Users/johnsmith/Downloads/chromedriver.exe`
2. What operating system are you running? From what I gather on Stack Overflow, it seems the error message you’re seeing may be triggered by the way the path to the chromedriver.exe file is set. Once I know what OS you’re running, I can see what changes I may need to make you the code

Let me know, and I’ll keep investigating this week.

**Comment by arminta7** [comment]
I didn't enter any of the optional info, so I didn't put the chromedriver.exe path. 

I am on MacOS Big Sur. I also don't have Chrome installed (I had to do the chromeisbad.com stuff because it was slowing my computer so badly). I didn't see Chrome in the code requirements so didn't think it would matter if I didn't want tags.

**Comment by nicrivard** [comment]
Ah! In that case, I think I've identified the root issue and fixed the code in [release v1.4](https://github.com/nicrivard/readwise2directory/releases/tag/v1.4). 

Please give it another go, and let me know if you come across any more issues!

**Comment by arminta7** [comment]
That worked, thanks!
